scale-tidb-using-tiup.md

```diff
@@ -359,17 +359,24 @@ tiup cluster display <cluster-name>
 
 ### 1. 根据 TiFlash 剩余节点数调整数据表的副本数
 
-在下线节点之前，确保 TiFlash 集群剩余节点数大于等于所有数据表的最大副本数，否则需要修改相关表的 TiFlash 副本数。
+1. 查询 TiFlash 副本数大于缩容后的 TiFlash 节点数的数据表。`tobe_left_nodes` 表示缩容后的 TiFlash 节点数。如果查询结果为空，可以开始执行缩容。如果查询结果不为空，则需要修改相关表的 TiFlash 副本数。
 
-1. 在 TiDB 客户端中针对所有副本数大于集群剩余 TiFlash 节点数的表执行：
+	{{< copyable "sql" >}}
+
+    ```sql
+    select * from information_schema.tiflash_replica where REPLICA_COUNT >  'tobe_left_nodes'
+    ```
+
+2. 对所有 TiFlash 副本数大于缩容后的 TiFlash 节点数的表执行：
 
     {{< copyable "sql" >}}
 
     ```sql
-    alter table <db-name>.<table-name> set tiflash replica 0;
+    alter table <db-name>.<table-name> set tiflash replica ‘tobe_left_nodes’;
     ```
```

Comment:
alter table <db-name>.<table-name> set tiflash replica ‘a new number of copies which should be less than obe_left_nodes ’;